website/docs/components/reveal/partials/accessibility/accessibility.md

```diff
@@ -4,6 +4,8 @@
 
 Reveal is conformant if the text does not change when the user interacts with the toggle button. In cases where the button text does change upon interaction, the toggle button is not conformant.
 
+For improved accessibility, ensure use of the `aria-describedby` attribute (see the API docs) to provide the toggle with a more accessible name.
```

Comment:
[suggestion] should this be an [Accessibility alert](https://hashicorp.atlassian.net/wiki/spaces/HDS/pages/3344269975/Markdown+Best+Practices#Accessibility)?